### PR TITLE
Feature/1058 permission detail tabs part 4

### PR DESCRIPTION
**Pull request by mimse**
## Description

Added that market roles the permission is included in

image

## References

- https://github.com/Energinet-DataHub/Titans/issues/14

### CODE REVIEW

File: libs/dh/admin/feature-permissions/src/lib/details/tabs/dh-admin-permission-market-roles.component.ts [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    LetModule,
+  ],
+})
+export class DhAdminPermissionMarketRolesComponent implements OnInit, OnChanges, OnDestroy {
+  @Input() selectedPermission: PermissionDto | null = null;
+  private apollo = inject(Apollo);
+  private trans: TranslocoService = inject(TranslocoService);
+
+  subscription!: Subscription;
+  marketRoles?: MarketRole[];
```
Comment (dzhavat) [inline libs/dh/admin/feature-permissions/src/lib/details/tabs/dh-admin-permission-market-roles.component.ts:79]: suggestion: can be initialized to an empty array.

File: libs/dh/admin/feature-permissions/src/lib/details/tabs/dh-admin-permission-market-roles.component.ts [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    WATT_TABLE,
+    WattEmptyStateModule,
+    TranslocoModule,
+    LetModule,
+  ],
+})
+export class DhAdminPermissionMarketRolesComponent implements OnInit, OnChanges, OnDestroy {
+  @Input() selectedPermission: PermissionDto | null = null;
+  private apollo = inject(Apollo);
+  private trans: TranslocoService = inject(TranslocoService);
```
Comment (dzhavat) [inline libs/dh/admin/feature-permissions/src/lib/details/tabs/dh-admin-permission-market-roles.component.ts:76]: nitpick: can be removed as it doesn't seem used.

File: libs/dh/admin/feature-permissions/src/lib/details/tabs/dh-admin-permission-roles.component.ts [hunk 3]
```diff
@@ -113,4 +128,9 @@ export class DhAdminPermissionRolesComponent implements OnInit, OnChanges, OnDes
       this.getPermissionQuery?.refetch({ id });
     }
   }
+
+  translateHeader = (columnId: string): string => {
+    const baseKey = 'admin.userManagement.permissionDetail.tabs.userRoles.columns';
+    return this.trans.translate(`${baseKey}.${columnId}`);
+  };
```
Comment (dzhavat) [inline libs/dh/admin/feature-permissions/src/lib/details/tabs/dh-admin-permission-roles.component.ts:135]: nitpick: unintentionally has sneaked in :)

Comment (mimse) [reply libs/dh/admin/feature-permissions/src/lib/details/tabs/dh-admin-permission-roles.component.ts]: https://github.com/Energinet-DataHub/greenforce-frontend/pull/1589/commits/ffe5f38eb1d959573b35edf79f19a6be9d0c190c

File: libs/dh/admin/feature-permissions/src/lib/details/tabs/dh-admin-permission-market-roles.component.ts (hunk 1 above)
Comment (mimse) [reply libs/dh/admin/feature-permissions/src/lib/details/tabs/dh-admin-permission-market-roles.component.ts]: https://github.com/Energinet-DataHub/greenforce-frontend/pull/1589/commits/ffe5f38eb1d959573b35edf79f19a6be9d0c190c

**Review by dzhavat — APPROVED** [review]

**Pull request merged**